app/src/main/java/com/infomaniak/swisstransfer/ui/components/SwissTransferAlertDialog.kt

```diff
@@ -0,0 +1,134 @@
+/*
+ * Infomaniak SwissTransfer - Android
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.swisstransfer.ui.components
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.infomaniak.swisstransfer.R
+import com.infomaniak.swisstransfer.ui.theme.Margin
+import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SwissTransferAlertDialog(
+    modifier: Modifier = Modifier,
+    @StringRes titleRes: Int,
+    @StringRes descriptionRes: Int,
+    onDismiss: () -> Unit,
+    onConfirmation: () -> Unit,
+    shouldEnableConfirmButton: () -> Boolean = { true },
+    content: @Composable (ColumnScope.() -> Unit)? = null,
+) {
+    BasicAlertDialog(
+        onDismissRequest = onDismiss,
```

Comment:
Shouldn't this one be renamed `onDismiss` too ?

Reply:
… this one is in the Android SDK 🤣 

Can you do a PR on the AOSP repository to rename it, please ? 😁

Reply:
Yeah sure 😂